modules/ztp-sno-du-configuring-lvms.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="lvms-configuring-lvms-on-sno_{context}"]
+= {lvms}
+
+You can dynamically provision local storage on {sno} clusters with {lvms-first}.
+The following YAML example configures the storage of the node to be available to {product-title} applications.
```

Comment:
> configures the storage of the node to be available

Not sure what that means. _Configures storage for the node...._ ?

Reply:
My understanding was that "The YAML config makes the storage of the node available to OCP apps."

Reply:
That make sense to me!